recipes/yt/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py --quiet install --single-version-externally-managed --record=record.txt
```

Comment:
Move to `script` in `build` section of `meta.yaml`. See notes. Then remove this file.